index.html

```diff
@@ -168,13 +168,13 @@ <h1 class="contact-title">Contact me</h1>
                     <p class="content-3">
                         If you have an application you are interested in developing, 
                         a feature that you need built or a project that needs coding. 
-                        I’d love to help with it.</p>
-                <div class="text-me">
-                    <input type="text" class="simple" placeholder="User name">
-                    <input type="email" class="simple" placeholder="Email">
-                    <textarea class="text-area" placeholder="comment"></textarea>
+                        i'd love to help with it.</p>
+                <form action="index.html" methop="post" class="text-me">
```

Comment:
- [ ] As the previous reviewer mentioned, It is required to add formspree service to your form to can deliver form requests in your email, now go to [formspree](https://formspree.io/) and register and deliver your id to enjoy this service.
your form will be like this `<form action="https://formspree.io/f/{form_id}" method="post">`
form_id: the id will recieve